pytracking_config.yaml

```diff
@@ -0,0 +1,32 @@
+
+ObjectTracker:
+        run_optical_flow: False # A flag if to run optical flow(using the matcher class)
+        use_orb: False # A flag if to use orb when optical flow falied
+        tracker_run_iter: 3 #  The number of times to run matcher between deep-tracker run.
```

Comment:
What happened if run_optical_flow is False?

Reply:
Then optical flow wont run and only the custom tracker